titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/KubeUtil.java

```diff
@@ -234,8 +238,13 @@ public static String getNodeIpV4Address(V1Node node) {
         annotations.put("containerInfo", encodedContainerInfo);
 
         if (includeJobDescriptor) {
-            String jobDescriptorJson = CommonObjectMappers.writeValueAsString(ObjectMappers.storeMapper(), job.getJobDescriptor());
-            annotations.put("jobDescriptor", StringExt.gzipAndBase64Encode(jobDescriptorJson));
+            JobDescriptor grpcJobDescriptor = GrpcJobManagementModelConverters.toGrpcJobDescriptor(job.getJobDescriptor());
```

Comment:
Does it mean this field is not processed yet by VK?

Reply:
correct